buildstockbatch/schemas/v0.3.yaml

```diff
@@ -0,0 +1,120 @@
+schema_version: enum('0.3')
```

Comment:
@nmerket Is there a reason this file is a '.yaml' and the rest in this repo are '.yml'?

Reply:
No reason.

Reply:
I take that back. The schemas are all `.yaml` There's no particular reason for it other than consistency. The end user doesn't interact with this file, so I don't feel a need to change it.